source/http/HttpConnection.cpp

```diff
@@ -130,6 +130,7 @@ namespace Aws
                 options.socket_options = &connectionOptions.SocketOptions.GetImpl();
                 options.on_setup = HttpClientConnection::s_onClientConnectionSetup;
                 options.on_shutdown = HttpClientConnection::s_onClientConnectionShutdown;
+                options.enable_read_back_pressure = connectionOptions.EnableReadBackPressure;
```

Comment:
oh, dammit. I didn't see this when we introduced in to aws-c-http. We already had a variable like this on streams. It used a different term that was closer to the terminology for HTTP/2 window stuff. It looked like this:

```
    /**
     * Set to true to manually manage the read window size.
     *
     * If this is false, the connection will maintain a constant window size.
     *
     * If this is true, the caller must manually increment the window size using aws_http_stream_update_window().
     * If the window is not incremented, it will shrink by the amount of body data received. If the window size
     * reaches 0, no further data will be received.
     */
    bool manual_window_management;
```

The effect of that variable was that, if you left it false (default) the user wouldn't have to call stream.UpdateWindow(), the connection would do it for you.

But now that we can totally turn off back-pressure, we should probably have the option on the connection, instead of the individual streams? I still like the term "manual window management" over "enable read back pressure", since that's the HTTP/2 terminology